otherlibs/systhreads/st_stubs.c

```diff
@@ -192,6 +194,7 @@ static void save_runtime_state(void)
   CAMLassert(This_thread != NULL);
   caml_thread_t this_thread = This_thread;
   this_thread->current_stack = Caml_state->current_stack;
+  this_thread->dls_state = Caml_state->dls_root;
```

Comment:
Should both `dls_state` and `dls_root` be renamed to the same thing, similar to other fields? One option is `tls_state` or `tls_root`.

Reply:
I feel like it's overkill to require a bootstrap just for an internal name like this. This is however just my opinion, possibly biased by the fact it's been years since I did a bootstrap on ocamlc :)